tangelo/linq/circuit.py

```diff
@@ -144,6 +146,16 @@ def is_mixed_state(self):
         """
         return "MEASURE" in self.counts
 
+    def draw(self):
+        """Method to output a prettier version of the circuit for use in jupyter notebooks that uses cirq SVGCircuit"""
+        # circular import
+        from tangelo.linq.translator.translate_cirq import translate_c_to_cirq
+        warnings.filterwarnings("ignore", message=".*findfont.*")
```

Comment:
This does not seem to work for me (I don't understand your pattern btw, I'm not sure why the dots are here).
Let's consider this a Cirq issue and remove this line as it does not work.